### PR TITLE
Update errorprone

### PR DESCRIPTION
**Pull request by anuraaga**
I'm appreciating errorprone finding when we are recording longs into double histograms (well, assuming we're not supposed to be doing that :P)

### CODE REVIEW

File: jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java [hunk 1]
```diff
@@ -27,7 +27,6 @@ public ContainerConfigurationHandler() {
   public void initializeMeter(Meter meter) {
     meter
         .upDownCounterBuilder(METRIC_NAME)
-        .ofDoubles()
```
Comment (anuraaga) [inline jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java:30]: @kittylyst Can you check out this PR, especially this line? There was intention of using doubles even though the value is a long

**Review by mateuszrzeszutek — APPROVED** [review]

File: jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java (hunk 1 above)
Comment (kittylyst) [reply jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java:30]: @anuraaga Should this value be a long? Is it not popssible to have fractional CPU counts?

**Review by kittylyst — APPROVED** [review]
Apart from that one comment about CPU counts, this looks good to me.

File: jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java (hunk 1 above)
Comment (anuraaga) [reply jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/container/ContainerConfigurationHandler.java:30]: We use `getLong` below to retrieve so it seems no. Unless we should get double?

**Review by trask — APPROVED** [review]
> I'm appreciating errorprone finding when we are recording longs into double histograms

ya those are some really good finds 🎉

**Review by jack-berg — APPROVED** [review]

**Pull request merged**